bin/helm-deploy

```diff
@@ -7,6 +7,16 @@ set -eo pipefail
 
 HELM_DEFAULT_TIMEOUT=300
 
+format_multiple_values_files() {
+  values_files="$1"
+  formatted_files=""
+  echo "${values_files}" | sed -n 1'p' | tr ',' '\n' | ( while read file_name; do
+    file="./deploy/${file_name}.values.yml,"
+    formatted_files="$formatted_files$file"
+  done
+  echo $formatted_files )
+}
```

Comment:
Did you intend $formatted_files to be inside the `()`helm

Reply:
its silly because I added the `()` just for this last line 17. Its interesting because the `while` loop is executed in a subshell so when the variable `$formatted_files` is set on line 15 it does not persist outside of the `while read` loop on line 17.  I found a solution to that here https://serverfault.com/a/259342